guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>Below are some examples for satellite imagery for the Landsat and Pleiades satellites along with the appropriate <code>raster_type_name</code> and <code>raster_type_params</code>:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>